File: sdkwork-ai-sdk-csharp/Api/ModerationsApi.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Ai.Http;
using Ai.Models;

namespace Ai.Api
{
    public class ModerationsApi
    {
        private readonly HttpClient _client;

        public ModerationsApi(HttpClient client)
        {
            _client = client;
        }

        /// <summary>
        /// Create moderation
        /// </summary>
        public async Task<ModerationResponse?> CreateModerationAsync(ModerationRequest body)
        {
            return await _client.PostAsync<ModerationResponse>(ApiPaths.AiPath("/moderations"), body);
        }
    }
}

File: sdkwork-ai-sdk-swift/Sources/API/ModerationsApi.swift
import Foundation

public class ModerationsApi {
    private let client: HttpClient
    
    public init(client: HttpClient) {
        self.client = client
    }

    /// Create moderation
    public func createModeration(body: ModerationRequest) async throws -> ModerationResponse? {
        let response = try await client.post(ApiPaths.aiPath("/v1/moderations"), body: body)
        return response as? ModerationResponse
    }
}
